common/src/benchmarking.rs

```diff
@@ -131,16 +132,24 @@ pub fn generate_wasm3(payload: Vec<u8>) -> Result<Vec<u8>, &'static str> {
     Ok(code)
 }
 
-pub fn set_program(program_id: H256, code: Vec<u8>, static_pages: u32) {
+pub fn set_program(program_id: H256, code: Vec<u8>, static_pages: WasmPageNumber) {
     let code_id = CodeId::generate(&code).into_origin();
+    let wasm_pages = (0..static_pages.0)
+        .map(WasmPageNumber)
+        .collect::<Vec<WasmPageNumber>>();
```

Comment:
No need to be so expliit.
```suggestion
        .collect::<Vec<_>>();
```